tests/test_sparse_embeddings.py

```
@@ -311,6 +336,127 @@ def test_disable_stemmer_behavior(disable_stemmer: bool) -> None:
+        "\u062f\u0627\u0646\u0634\u06af\u0627\u0647 \u062a\u0647\u0631\u0627\u0646",
+        "\u0627\u06cc\u0646 \u0645\u062a\u0646 \u0641\u0627\u0631\u0633\u06cc \u0627\u0633\u062a",
+    ] * 3
+
+    parallel_embeddings = list(model.embed(documents, batch_size=1, parallel=2))
+    single_process_embeddings = list(model.embed(documents, batch_size=1, parallel=None))
+
+    assert len(parallel_embeddings) == len(single_process_embeddings) == len(documents)
+    for parallel_embedding, single_process_embedding in zip(
+        parallel_embeddings,
+        single_process_embeddings,
+    ):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)